src/SFA.DAS.EAS.Portal.Worker.UnitTests/SFA.DAS.EAS.Portal.Worker.UnitTests.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It'd be nice to have all unit tests in the single `SFA.DAS.EAS.Portal.UnitTests` project, rather than a unit test project for each assembly under test. This should keep the number of projects in the solution down, which will help with solution load and CI/CD times.

Reply:
This has been actioned

Reply:
👍